src/modules/show_score.js

```diff
@@ -0,0 +1,13 @@
+const fill = (data) => {
+  const containScores = document.querySelector('.containScores');
+  while (containScores.firstChild) {
+    containScores.removeChild(containScores.firstChild);
+  }
+  data.forEach((m) => {
+    const li = document.createElement('li');
+    li.innerHTML = `${m.user} : ${m.score}`;
+    containScores.appendChild(li);
+  });
+};
+
```

Comment:
- [ ] I see you made functionality here for it. when I was trying it wasn't showing any of getting requests or any score even tho your post request works fine 
It was weird it is not showing anything so I am looking to hear back from you it might be my installation or something because I am sure you wouldn't submit if it wasn't showing

![image](https://user-images.githubusercontent.com/87689547/157594039-f293ae6e-b3fb-419e-a7b0-68b12af3d17f.png)

Reply:
Thank you friend.